datadog_checks_dev/datadog_checks/dev/plugin/pytest.py

```diff
@@ -130,6 +131,19 @@ def dd_environment_runner(request):
         pytest.exit(message)
 
 
+@pytest.fixture(scope='session', autouse=True)
+def dd_environment_skipper(request):
+    # Skip the runner if the environment variable is specified
+    do_skip = os.getenv(SKIP_ENVIRONMENT) == 'true'
+    if not do_skip:
+        return
+    # Since the scope is `session` there should only ever be one definition
+    fixture_def = request._fixturemanager._arg2fixturedefs[E2E_FIXTURE_NAME][0]
+
+    # Make the underlying function a no-op
+    fixture_def.func = lambda: None
```

Comment:
Maybe we can add this new logic to the existing `dd_environment_runner`

Reply:
Done!